python_modules/dagster/dagster/_core/storage/asset_check_execution_record.py

```diff
@@ -22,9 +24,36 @@ class AssetCheckExecutionResolvedStatus(enum.Enum):
     SKIPPED = "SKIPPED"  # the run finished, didn't fail, but the check didn't execute
 
 
-class AssetCheckExecutionRecord(NamedTuple):
-    id: int
-    run_id: str
-    status: AssetCheckExecutionRecordStatus
-    evaluation_event: Optional[EventLogEntry]
-    create_timestamp: float
+class AssetCheckExecutionRecord(
+    NamedTuple(
+        "_AssetCheckExecutionRecord",
+        [
+            ("id", int),
```

Comment:
Kinda wish this was `storage_id`, but NBD?

Reply:
i'll refactor that